### PR TITLE
Add initial version of must-gather image

### PR DESCRIPTION
**Pull request by slintes**
```release-note
Added initial version of must gather image
```

### CODE REVIEW

File: must-gather/collection-scripts/gather [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+. ${DIR_NAME}/version
+echo "node-maintenance-operator/must-gather" > /must-gather/version
+version >> /must-gather/version
+
+# get namespace of node-maintenance-operator
+NMO_NAMESPACE=$( oc get pods -A -l name='node-maintenance-operator' -o=jsonpath='{.items[0].metadata.namespace}{"\n"}' )
+# fallback to default
+[ -z "${NMO_NAMESPACE}" ] && NMO_NAMESPACE="openshift-operators"
+
+# TODO get nmo logs
```
Comment (razo7) [inline must-gather/collection-scripts/gather:16]: Why `TODO`? Does something is missing?

File: must-gather/collection-scripts/gather [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# generate /must-gather/version file
+DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. ${DIR_NAME}/version
+echo "node-maintenance-operator/must-gather" > /must-gather/version
+version >> /must-gather/version
+
+# get namespace of node-maintenance-operator
+NMO_NAMESPACE=$( oc get pods -A -l name='node-maintenance-operator' -o=jsonpath='{.items[0].metadata.namespace}{"\n"}' )
+# fallback to default
+[ -z "${NMO_NAMESPACE}" ] && NMO_NAMESPACE="openshift-operators"
```
Comment (razo7) [inline must-gather/collection-scripts/gather:14]: Shouldn't be the same namespace of RHWA product, `rhwa-workload-availability`?

File: must-gather/collection-scripts/gather (hunk 1 above)
Comment (slintes) [reply must-gather/collection-scripts/gather:16]: we might want to gather NMO logs, but I didn't have time to check how to do that. We can add that as a follow up. It's more important to have something ready for the ds builds quickly.

**Pull request merged**